lib/reports/subscriber_subscriptions.rb

```diff
@@ -0,0 +1,44 @@
+module Reports
+  class SubscriberSubscriptions
+    def initialize(email_addresses)
+      @email_addresses = email_addresses
+    end
+
+    def self.call(email_addresses:)
+      new(email_addresses).call
```

Comment:
It might be better to do `self.call(*args)` here so if the arguments to `initialize` changes, we won't need to update it in two places.

Reply:
👍 updated in bde568a